pkg/discovery/discovery.go

```diff
@@ -361,23 +361,34 @@ func (d *Discoverer) discoverVolumesAtPath(class string, config common.MountConf
 				discoErrors = append(discoErrors, fmt.Errorf("path %q of filesystem mode cannot be used to create block volume", filePath))
 				continue
 			}
-			// TODO(mauriciopoppe): skipping the mountpoint verification in Windows, unlike Linux there's no /proc/mounts file to check
 			if runtime.GOOS == "windows" {
 				// the path to use is the one sent to CSI Proxy and must be in the context of the host
 				// not the container, filePath is in the context of the container
-				volumePathInHost := filepath.Join(config.HostDir, file)
-				capacityByte, err = d.VolUtil.GetFsCapacityByte(volumePathInHost)
+				volumePathInHostContext := filepath.Join(config.HostDir, file)
```

Comment:
Is it worth it to make this logic part of the utils? I'm not sure if there's much benefit to creating the vol utils interfaces but still have to have OS checks in the callers.

Reply:
agreed, I moved some of the code to verify the `/proc/mounts` file to the linux VolumeUtil impl, as a result there's only a single path in the discovery code

Reply:
Thanks, can you also apply this pattern to the deleter too?

Reply:
sure, I updated it to receive both the hostpath & mountpath